app/form_models/copy_vacancy_form.rb

```diff
@@ -25,12 +31,23 @@ def initialize(vacancy:)
   end
 
   def apply_changes!(params = {})
+    assign_attributes(params.extract!(:expiry_time_hh, :expiry_time_mm, :expiry_time_meridiem))
     vacancy.assign_attributes(params)
+    vacancy.expiry_time = compose_expiry_time
     vacancy
   end
 
   private
 
+  def compose_expiry_time
```

Comment:
I think this is a duplicate too of the code in the other form?  Did you think about removing that duplication?